packages/mobile/e2e/scripts/fund-e2e-accounts.ts

```diff
@@ -14,11 +14,16 @@ const kit = ContractKit.newKitFromWeb3(web3)
 
 // Throw error if balance is low
 const balanceError = async (address = valoraE2ETestWallet, minBalance = 10) => {
-  let balanceObject = await getBalance(address)
-  for (const balance in balanceObject) {
-    if (balanceObject[balance] < minBalance) {
-      throw new Error(`Balance of ${address} is below ${minBalance}`)
+  try {
+    let balanceObject = await getBalance(address)
+    for (const balance in balanceObject) {
+      if (balanceObject[balance] < minBalance) {
+        throw new Error(`Balance of ${address} is below ${minBalance}`)
```

Comment:
nice! i was wondering if you think it'd be a good idea to also add some text about how to fund the accounts to resolve the error? i am imagining myself seeing this error and wanting to resolve it to keep the pipeline going. it'd be super helpful to see something like `Balance of ${address} is below ${minBalance}, follow these instructions (on the readme? notion?) to top up the balance`?

Reply:
+1